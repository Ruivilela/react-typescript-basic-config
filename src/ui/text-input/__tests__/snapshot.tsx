import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from '..';

describe('TextInput', () => {
  it('should render', () => {
    const component = renderer.create(<TextInput/>);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
