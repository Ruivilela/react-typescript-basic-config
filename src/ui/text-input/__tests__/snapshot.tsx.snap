// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput should render 1`] = `
<input
  onChange={[Function]}
  value=""
/>
`;
